client/v3/config.go

```diff
@@ -90,3 +93,103 @@ type Config struct {
 
 	// TODO: support custom balancer picker
 }
+
+type ClientConfig struct {
+	Endpoints        []string      `json:"endpoints"`
+	RequestTimeOut   time.Duration `json:"request-timeout"`
+	DialTimeout      time.Duration `json:"dial-timeout"`
+	KeepAliveTime    time.Duration `json:"keepalive-time"`
+	KeepAliveTimeout time.Duration `json:"keepalive-timeout"`
+	Scfg             *SecureCfg    `json:"secure-cfg"`
+	Acfg             *AuthCfg      `json:"auth-cfg"`
+}
+
+type SecureCfg struct {
```

Comment:
```suggestion
type SecureConfig struct {
```
Let's be consistent in naming. If we have `ClientConfig` then let's use `SecureConfig` etc.

Reply:
fixed & thanks.